src/lib/coda_base/external_transition.ml

```diff
@@ -5,31 +5,23 @@ module type S = sig
 
   module Staged_ledger_diff : sig
     type t [@@deriving bin_io, sexp]
-
-    module Verified : sig
-      type t [@@deriving sexp, bin_io]
-    end
   end
 
   type t [@@deriving sexp, bin_io, compare, eq]
 
   module Verified : sig
     type t [@@deriving sexp, bin_io, compare, eq]
 
-    val create :
-         protocol_state:Protocol_state.value
-      -> protocol_state_proof:Proof.t
-      -> staged_ledger_diff:Staged_ledger_diff.Verified.t
-      -> t
-
     val protocol_state : t -> Protocol_state.value
 
     val protocol_state_proof : t -> Proof.t
 
-    val staged_ledger_diff : t -> Staged_ledger_diff.Verified.t
+    val staged_ledger_diff : t -> Staged_ledger_diff.t
   end
 
-  val forget : Verified.t -> t
+  val to_verified : t -> Verified.t
```

Comment:
this is suspicious

Reply:
please call this `to_verified_unsafe` and return 

```ocaml
t -> [`I'm_certain_this_is_safe_and_I_explained_it_in_a_comment of Verified.t]
```

Reply:
This is now the only way to convert an external transition into a verified one. There is no longer any kind of mutation that occurs, they are just identity types. This is the same exact way it used to work in the ledger builder controller when we modeled verified and unverified information.